SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# pygeoapi Security Policy
+
+## Supported Versions
+
+Security/vulnerability reports **should not** be submitted through GitHub issues or public discussions, but instead please send your report 
+to **tomkralidis nospam @ gmail.com** (remove the blanks and 'nospam').  
```

Comment:
You can add me as alternate contact so we can reach the minimum number in my opinion. However, it's likely a unique mailing list address dedicated to the security reports would be better

Reply:
@francbartoli I've updated the PR with a more generic email address.  We can merge once the address is setup.